universal-login-react/src/ui/Onboarding/Onboarding.tsx

```diff
@@ -7,6 +7,8 @@ import {getStyleForTopLevelComponent} from '../../core/utils/getStyleForTopLevel
 import {ReactModalContext, ReactModalProps, ReactModalType, TopUpProps} from '../../core/models/ReactModalContext';
 import {useModalService} from '../../core/services/useModalService';
 
+export const DEPLOYMENT_INFO = 'It takes time to register your username and deploy your wallet. In order to do so, we need to create a transaction and wait until the Ethereum blockchain validates it...';
```

Comment:
move to core/constants?

Reply:
Do we have other strings like this in core/constants?